examples/multiple_tables/main.tf

```diff
@@ -24,6 +24,7 @@ module "bigquery" {
   project_id                  = var.project_id
   location                    = "US"
   tables                      = var.tables
+  external_tables             = var.external_tables
```

Comment:
There's no need to make this a variable on the example (that's unnecessary complexity, though I know the existing examples use this approach). Instead you should just hard-code the value in the example directly.

Reply:
I agree - I had the same thought, but opted to follow the examples that were already in place. I removed the variable in 
20466e0  and took the liberty of doing the same for the `tables` variable in 
482d25d